gnome-initial-setup/pages/account/gis-account-page-local.ui

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <interface>
-  <!-- interface-requires gtk+ 3.0 -->
+  <!-- interface-requires gtk+ 3.30 -->
```

Comment:
Did you get this change from Glade or did you make it manually? Generally we avoid touching unrelated areas of the code.